### ISSUE TITLE
Checkboxes aren't accessible or localized

### ISSUE THREAD

**Issue by amyjko** [wordplaydev/wordplay#192]
## Expected behavior

Checkboxes should have localized labels, proper `aria-` tags, and follow other accessibility guidelines:

https://www.a11ymatters.com/pattern/checkbox/

## Actual behavior

Checkboxes are just simple inputs with no labels. Enable a screen reader, and they'll just be described as a checkbox and nothing more. They should have localized descriptions.

## Fix

* Add a required description field in `Checkbox.svelte` 
* Allow checkboxes to have an optional label
* Add necessary localization strings for all checkboxes
